TaxiiFeed: fixed an issue where score was not handled properly (#27213)
demisto/content: Packs/FeedTAXII/ReleaseNotes/1_1_27.md

#### Integrations
##### TAXII 2 Feed

> - Fixed an issue where the score was not handled properly on "update custom fields" mode.

Reviewer: ```suggestion
- Fixed an issue where indicators score were not handled properly when enabling the *update custom fields* feature.
```
